src/lib.rs

```diff
@@ -63,6 +63,13 @@ pub use settings::{opts_with_clap, RunMode};
 pub fn run(init: impl FnOnce(&mut VirtualMachine) + 'static) -> ExitCode {
     env_logger::init();
 
+    #[cfg(target_os = "wasi")]
+    {
```

Comment:
Thank you for explanation. I think adding this feature is reasonable. Because this is neither a WASI feature not a Python feature, can we just put a note that this is our own convenience feature?

```suggestion
    {
        // NOTE: This is not a WASI convention. But it will be convenient since POSIX shell always defines it.
```

Reply:
Oh yea sure. I can add a note about this. Thanks for being open to this 🙏 I agree with you that it's a convenience feature from RustPython in this case.

Reply:
Done.